CromIAM/src/main/scala/cromiam/server/config/CromIamServerConfig.scala

```diff
@@ -1,28 +1,37 @@
 package cromiam.server.config
 
 import akka.http.scaladsl.settings.ServerSettings
+import cats.instances.option._
 import cats.syntax.apply._
+import cats.syntax.traverse._
 import cats.syntax.validated._
 import com.typesafe.config.{Config, ConfigFactory}
-import common.validation.ErrorOr.ErrorOr
+import common.validation.ErrorOr._
+import common.validation.Validation._
+import cromiam.server.config.CromIamServerConfig._
+import cromwell.cloudsupport.gcp.GoogleConfiguration
+import cromwell.cloudsupport.gcp.auth.GoogleAuthMode
+import net.ceedubs.ficus.Ficus._
 
 import scala.collection.JavaConverters._
 import scala.util.{Failure, Success, Try}
-import cromiam.server.config.CromIamServerConfig._
 
 final case class CromIamServerConfig(cromIamConfig: CromIamConfig,
                                      cromwellConfig: ServiceConfig,
                                      samConfig: ServiceConfig,
+                                     whitelistAuthModeOption: Option[GoogleAuthMode],
```

Comment:
I see why you were asking about this now. This (and the relevant portion in the conf) seems like it should be part of the sam config block*. Making that change would imply that `ServiceConfig` is no longer the right type for `samConfig`, but instead an e.g. `SamConfig` type which contains a `ServiceConfig`

* if I'm correct in believing that it's only used by the sam client in cromiam